include/nogdb_types.h

```diff
@@ -197,12 +200,15 @@ namespace nogdb {
 
     class Record {
     public:
+
+        using StringToBytesMap = std::map<std::string, Bytes>;
```

Comment:
How about 'PropertyData'? having 'nogdb::Record::PropertyData' may be nice for developers. 'StringToBytesMap' is quite specific and it doesn't mean anything. Let me know what you think.

Reply:
I think 'StringToBytesMap' gives a clear definition of what type it really is and how to use it. With 'ProptertyData', developers have to look up to its real definition.

Reply:
'PropertyToBytesMap' sounds great. I probably go with it.